microsoftgraph/microsoft-graph-explorer-v4
# Fix: sample query sample body

## Overview

Noticed that the sample body was not displayed as HTML in the editor.
Enhanced code to get and display data in the DetailList to a reusable function

### Notes

Refactoring has been done to generate groups from a list into a reusable function so that the logic is handled in one place. 


## Testing Instructions

* Log in to graph explorer
* search for "create page" in the samples
* Click the sample
* Notice the HTML is displayed
* Include test case, and expected output
